tests/httpServer/server.test.ts

```diff
@@ -76,6 +77,12 @@ describe("httpserver-tests", () => {
     expect(response.text).toBe("hello alice");
   });
 
+  test("post-test-custom-body", async () => {
+    const response = await request(testRuntime.getHandlersCallback()).post("/testpost").set('Content-Type', 'application/custom-content-type').send(JSON.stringify({ name: "alice" }));
+    expect(response.statusCode).toBe(200);
+    expect(response.text).toBe("hello alice");
+  });
```

Comment:
Can you also check the path with an ignored content type?

Reply:
Added, but it's more a test of Koa than of us